contracts/Verifier.sol

```diff
@@ -134,17 +134,33 @@ contract VerifierContract {
     }
 
     function evalPolyAt(uint[] _p, uint _x) internal returns (uint) {
-        return 0;
+        uint out = 0;
+        uint powerOfX = 1;
+
+        for (uint i = 0; i < _p.length; i++) {
+            out = out.add(powerOfX.mul(_p[i]));
+            powerOfX = powerOfX.mul(_x).div(MODULUS);
```

Comment:
it is not `.div(MODULUS)` but `% MODULUS` ?

Reply:
oh my